patroni/config.py

```diff
@@ -541,7 +541,8 @@ def _set_section_values(section: str, params: List[str]) -> None:
         _set_section_values('postgresql', ['listen', 'connect_address', 'proxy_address',
                                            'config_dir', 'data_dir', 'pgpass', 'bin_dir'])
         _set_section_values('log', ['type', 'level', 'traceback_level', 'format', 'dateformat', 'static_fields',
-                                    'max_queue_size', 'dir', 'mode', 'file_size', 'file_num', 'loggers'])
+                                    'max_queue_size', 'dir', 'mode', 'file_size', 'file_num', 'loggers',
+                                    'deduplicate_heartbeat_logs'])
```

Comment:
The value also should be converted from str to bool.
To do that you need to add it to https://github.com/patroni/patroni/blob/b4eab4897161eb15b37c6995991d76bf56de9ff7/patroni/config.py#L552-L553

Reply:
There is also config validator that needs to be updated: https://github.com/patroni/patroni/blob/b4eab4897161eb15b37c6995991d76bf56de9ff7/patroni/validator.py#L991-L1005